src/planner.c

```diff
@@ -373,7 +373,7 @@ is_hypertable_chunk_dml(PlannerInfo *root, RelOptInfo *rel, Index rti, RangeTblE
 		if (parent_oid != InvalidOid && rte->relid != parent_oid)
 		{
 			Cache *hcache = ts_hypertable_cache_pin();
-			Hypertable *parent_ht = ts_hypertable_cache_get_entry(hcache, parent_oid);
+			Hypertable *parent_ht = ts_hypertable_cache_get_entry_missing_ok(hcache, parent_oid);
```

Comment:
I think this and surrounding lines should be replaced with `ts_is_hypertable()`

Reply:
I am not sure if such change is within the scope of the PR.

Reply:
Replaced with the suggestion.